src/ape/pytest/contextmanagers.py

```diff
@@ -75,7 +80,11 @@ def _check_dev_message(self, exception: ContractLogicError):
         if offending_source is None or offending_source.line_start is None:
             raise AssertionError("Could not find line that caused revert.")
 
-        assertion_error_prefix = f"Expected dev revert message '{self.dev_message}'."
+        assertion_error_prefix = "Expected dev revert message '{}.'".format(
```

Comment:
This is probably just a personal preference, but I find f-strs more readable than `.format()` strings.

```python
message = self.dev_message.pattern if isinstance(self.dev_message, re.Pattern) else self.dev_message
assertion_error_prefix = f"Expected dev revert message '{message}.'"
```